test-app/package.json

```diff
@@ -76,6 +76,7 @@
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^7.6.0",
     "ember-auto-import": "^2.5.0",
+    "ember-browser-services": "^4.0.4",
```

Comment:
this works around an issue in embroider which will be fixed in the next release -- ultimately though, I think we may want to make this a peer _eventually_ anyway, depending on what our components end up doing -- only time will tell though.